### PR TITLE
Add mapping linter to check for duplicate machinery stacked on same tile

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
This prevents stuff like:
- #79390
- #79401
- #79399
- #79400
- #79402
- #79403 
- #79405 
- #79406 
- #79407 
- #79408 
- #79409 
- #79410 
- #79477

This has happened quite a few times in the past and it'd be a good idea to have a linter catch this common mistake. After digging through the Python code for the mapping linter, it appears the linter wasn't correctly identifying two duplicate objects.  I tweaked the code to fix this.

CC @tgstation/maptainers 

## Why It's Good For The Game
Better mapping consistency.

## Changelog
:cl:
fix: Fix mapping linter not identifying duplicate blacklisted objects on a turf
code: Add a mapping linter to check for stacked machinery
/:cl:

### CODE REVIEW

File: tools/maplint/lints/multiple_computers.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/obj/machinery/computer:
```
Comment (JohnFulpWillard) [inline tools/maplint/lints/multiple_computers.yml:1]: Is there any instance where we want several ``/obj/machinery`` stacked ontop of eachother?

Comment (timothymtorres) [reply tools/maplint/lints/multiple_computers.yml]: Applied with 57a3784a354cf7438d018e395ad5905204759186

**Review by JohnFulpWillard — APPROVED** [review]

**Comment by iwishforducks** [comment]
https://youtu.be/Amy_RWpU8t4

**Comment by JohnFulpWillard** [comment]
This ready now?

**Comment by timothymtorres** [comment]
Yep, all the relevant mapping PRs are merged.

**Pull request merged**